config/routes.rb

```diff
@@ -4,23 +4,34 @@
 
   # You can have the root of your site routed with "root"
   root 'tasks#index'
-  post '/' => 'tasks#index'
-
   # Example of regular route:
-  get 'show/:id' => 'tasks#show'
+  get 'tasks/show/:id' => 'tasks#show'
```

Comment:
so much more routing practice now. I am definitely interested in keeping an eye on our custom routes for bEtsy to make sure we keep the implementation clean.